content/writing/markdown.md

```diff
@@ -61,3 +61,32 @@ will both render
 3. item 3
 
 The second way is more easy to manage when you have to reorder your list, but to ensure a better readability of the Markdown files when not rendered, we use the correct numbers for our list items.
+
+## Callout
+Callouts are a great method to attract additional attention to specific ideas or to make it obvious that some information is optional or only important in certain situations.
+
+Here is an example of how you to write a callout:
+
+```
```

Comment:
I would remove that since we list them we example rigth after

Reply:
I wanted to show an example of how it is written, most people wouldn't know the syntax used to achieve that

Reply:
We give them all examples right after

Reply:
Oh my bad, they are visual!